website/docs/docs/quickstarts/overview.md

```diff
@@ -0,0 +1,50 @@
+---
+title: About quickstarts
+id: overview
+sidebar_label: "About quickstarts"
+description: "Quickly create your first dbt project with SQL."
```

Comment:
I'm not sure how to add this eloquently, but the quickstart goes from creating the project to running it, which might be nice to capture in the description for Google searches. Nothing fancy, we're just showing you how to create and run your new project!
```suggestion
description: "Quickly create and run your first dbt project with SQL."
```